charts/asset-manager/templates/worker-deployment.yaml

```diff
@@ -20,25 +20,30 @@ spec:
         app: {{ .Release.Name }}-worker
     spec:
       securityContext:
+        runAsUser: 1001
+        runAsGroup: 1001
         seccompProfile:
           type: RuntimeDefault
-        # Asset Manager needs to run as 2899 since it shares the same NFS volume with
-        # EC2 counterpart
-        runAsUser: 2899
-        runAsGroup: 2899
       containers:
         - name: app
           image: "{{ required "A valid .Values.appImage.repository entry required!" .Values.appImage.repository }}:{{ .Values.appImage.tag }}"
           imagePullPolicy: {{ .Values.appImage.pullPolicy | default "Always" }}
+          command: ["bundle"]
+          args: ["exec", "sidekiq", "-C", "config/sidekiq.yml"]
           ports:
             - name: metrics
               containerPort: {{ .Values.metricsPort }}
-          command: ["bundle"]
-          args: ["exec", "sidekiq", "-C", "config/sidekiq.yml"]
+          volumeMounts:
+            - name: asset-manager-efs
+              mountPath: &uploads-path /mnt/asset-manager
```

Comment:
Instead of doing both the moving of the YAML block _and_ the change to the mount directory in the same commit, can we split it out to make it easier to follow/review?